source/andThen.js

```diff
@@ -14,7 +14,7 @@ import _assertPromise from './internal/_assertPromise';
  * @sig (a -> (Promise e b)) -> (Promise e a) -> (Promise e b)
  * @param {Function} onSuccess The function to apply. Can return a value or a promise of a value.
  * @param {Promise} p
- * @return {Promise} The result of calling `p.then(onSuccess)`
+ * @return {Promise} The result of calling `p.andThen(onSuccess)`
```

Comment:
Oops.  All the other `then`s were correctly changed to `andThen`s.  But this one shouldn't be.